docs/src/components/DocLayout/primitives/StyledMobileOutdent.tsx

```diff
@@ -13,6 +13,7 @@ const StyledMobileOutdent = styled.div`
 
   > *:last-child {
     flex: 1;
+    margin-bottom: 3rem;
```

Comment:
Just a reminder, I think this gap is unnecessary. My suggestion is to remove the margin.

<img width="605" alt="Screen Shot 2021-09-09 at 19 17 29" src="https://user-images.githubusercontent.com/471278/132732409-761876c2-7afd-4648-92a0-2235b88dd9af.png">